.github/dependabot.yml

```diff
@@ -10,7 +10,9 @@ updates:
       - "dependencies"
       - "github-actions"
 
-  # Python dependencies
+  # Python dependencies (via pyproject.toml)
+  # Note: Using pip ecosystem as dependabot doesn't support uv yet
+  # It will read from pyproject.toml and update dependencies there
   - package-ecosystem: "pip"
```

Comment:
I am a bit nervous about changing dependencies, since some (e.g. scikit-learn) may affect the experimental results.

I guess two options are:
1. Remove dependapot
2. Tag the branch now as the "ICML branch" or something similar, and keep developing.

Probably (2) is the better option anyway? I'm not too familiar with dependabot; is best practice just to accept all the PRs it opens?